rust/src/schema.rs

```diff
@@ -22,6 +23,11 @@ impl SchemaTypeStruct {
     pub fn get_fields(&self) -> &Vec<SchemaField> {
         &self.fields
     }
+
+    pub fn to_json(&self) -> Value {
```

Comment:
I don't think this method is needed, `Deserialize` trait + serde_json should handle this automatically for us. So you should be able to do something like this:

```rust
let s = SchemaTypeStruct{};
let json_s: serde_json::Value = s;
```